src/resources/action-map/filter.js

```diff
@@ -111,7 +111,7 @@ module.exports = {
   }),
   search: (l, r) => ({
     bool: {
-      filter: r.toLowerCase().split(/[^0-9a-z]+/)
+      filter: r.toLowerCase().split(/[^a-zA-Z0-9À-ÖØ-öø-ÿ]+/)
```

Comment:
This helps to keep the diacritics and graphemes in the search terms instead of removing them.

Before the function was splitting the search terms:
```
 -  "query": {
      -    "bool": {
      -      "filter": [
      -        {
      -          "bool": {
      -            "filter": [
      -              {
      -                "wildcard": {
      -                  "name": {
      -                    "value": "cr*"
      -                  }
      -                }
      -              }
      -              {
      -                "wildcard": {
      -                  "name": {
      -                    "value": "me*"
      -                  }
      -                }
      -              }
      -              {
      -                "wildcard": {
      -                  "name": {
      -                    "value": "brul*"
      -                  }
      -                }
      -              }
      -              {
      -                "wildcard": {
      -                  "name": {
      -                    "value": "e*"
      -                  }
      -                }
      -              }
      -            ]
      -          }
      -        }
      -      ]
      -    }
      -  }
```

The second part of this would be to enable ES to match the results for `crème` and creme no matters how the user is searching